pmlc/dialect/stdx/tests/i1StorageToi8.mlir

```diff
@@ -0,0 +1,30 @@
+// RUN: pmlc-opt -stdx-i1-storage-to-i8 %s | FileCheck %s
+
+module {
+  func @eltwise_add(%arg0: memref<10x20xf32>, %arg1: memref<10x20xf32>) {
+    %cst = constant 0.000000e+00 : f32
+    %0 = alloc() : memref<10x20xi1>
+    %c0 = constant 0 : index
+    %c10 = constant 10 : index
+    %c1 = constant 1 : index
+    loop.for %arg2 = %c0 to %c10 step %c1 {
+      %c0_3 = constant 0 : index
+      %c20 = constant 20 : index
+      %c1_4 = constant 1 : index
+      loop.for %arg3 = %c0_3 to %c20 step %c1_4 {
+        %1 = load %arg0[%arg2, %arg3] : memref<10x20xf32>
+        %2 = cmpf "olt", %1, %cst : f32
+        store %2, %0[%arg2, %arg3] : memref<10x20xi1>
+		%3 = load %0[%arg2, %arg3] : memref<10x20xi1>
+		store %3, %0[%arg2, %arg3] : memref<10x20xi1>
+      }
+    }
+    return
+  }
+  // CHECK: %{{.*}} = alloc() : memref<10x20xi8>
```

Comment:
So it's a matter of style and both are allowable, but I tend to prefer putting the check lines 'inline' with the code, so for example, you would have the alloc, and then the CHECK of the modified alloc, and then the store and the check of the modified store, etc.  Filecheck (which lit runs) doesn't actually care about the placement of the check lines, just the order, and I find the inlined version easier to read.